setup.py

```diff
@@ -18,7 +18,7 @@
     author="Yoshiki Vazquez-Baeza",
     author_email="yoshiki@ucsd.edu",
     description="Display ordination plots",
-    license="bsd",
+    license='BSD-3-Clause',
```

Comment:
So we would still have been missing the `3-clause` which was the part conda-forge was worried about. Given [this](https://github.com/conda-forge/staged-recipes/pull/1071#discussion_r71085754) comment, it did not sound like that have a particular ontology. I feel like it makes sense to use the SPDX ID, but I would be fine with `BSD 3-Clause` also.